src/java/org/apache/cassandra/service/paxos/v1/AbstractPaxosCallback.java

```diff
@@ -17,6 +17,8 @@
  */
 package org.apache.cassandra.service.paxos.v1;
 
+import org.apache.cassandra.transport.Dispatcher;
+import org.apache.cassandra.utils.Clock;
 import org.apache.cassandra.utils.concurrent.CountDownLatch;
```

Comment:
nit: All 3 of these imports should move to their places below?

Reply:
interesting; not sure why my intellij has moved them